2682. Find the Losers of the Circular Game/solution.py

```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # taken the number of players, n, and a constant, k, it returns the losers of a circular game. player 1 starts with the ball,
```

Comment:
Is there a typo in "taken"? I think it should be "given".

Reply:
I did intentionally do "taken", but it really should be "given.